aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -429,7 +429,7 @@ impl AptosVM {
                     Some(sender) => vec![sender, *execute_as],
                 };
                 let remapped_script =
-                    if aptos_version < aptos_types::on_chain_config::DIEM_VERSION_2 {
+                    if aptos_version < aptos_types::on_chain_config::APTOS_VERSION_2 {
                         None
                     } else {
                         script_to_script_function::remapping(script.code())
```

Comment:
same here

Reply:
can you make an issue to review all APTOS_VERSION shits and remove them?

Reply:
https://github.com/aptos-labs/aptos-core/issues/366